src/ingestor.ts

```diff
@@ -12,208 +12,218 @@ import { storeMsg } from './helpers/highlight';
 import log from './helpers/log';
 import { capture } from '@snapshot-labs/snapshot-sentry';
 import { flaggedIps } from './helpers/moderation';
+import { timeIngestorProcess } from './helpers/metrics';
 
 const NAME = 'snapshot';
 const VERSION = '0.1.4';
 const broviderUrl = process.env.BROVIDER_URL || 'https://rpc.snapshot.org';
 
 export default async function ingestor(req) {
-  const body = req.body;
+  const success = 0;
+  const type = '';
```

Comment:
We are not changing these anywhere? changes automatically?

Reply:
fixed, merge conflict gone wrong